test/fs.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const metatests = require('metatests');
+const common = require('..');
+
+metatests.test('create 2 extra empty directories', test => {
+  common.mkdirp('test/example1/example2');
```

Comment:
You should remove testing dir if it exists, that create it using `mkdirp`, check that it is actually created, and then you should it.